packages/table/src/table.tsx

```diff
@@ -540,56 +526,28 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
                 ref={this.setRootTableRef}
                 onScroll={this.handleRootScroll}
             >
-                <TableQuadrant
+                <TableQuadrantStack
```

Comment:
I don't think it's so bad—it's an ordered stack of quadrants! Other options:

`TableQuadrants`
`TableQuadrant...Chimney`

![image](https://user-images.githubusercontent.com/443450/29043358-b52afc72-7b6f-11e7-80d7-e895d943b887.png)